client/layout/masterbar/logged-in.jsx

```diff
@@ -125,6 +127,27 @@ class MasterbarLoggedIn extends Component {
 		}
 	};
 
+	setupReaderPositionObserver() {
+		if ( this.props.sectionName !== 'home' && this.props.sectionGroup !== 'sites-dashboard' ) {
```

Comment:
The tooltip only shows on these pages, so I'm limiting the observer to these pages. Also, E2E tests on other pages were breaking, though I'm not 100% sure why.

Reply:
Limiting to those pages is a good idea.

Reply:
Makes sense to me!

Reply:
Thank you!